web/src/routes/admin/users/[id]/+page.svelte

```diff
@@ -69,6 +71,21 @@
   let canResetPassword = $derived($authUser.id !== user.id);
   let newPassword = $state<string>('');
 
+  let lastUploadText = $derived(
```

Comment:
We can also use this option which is even more accurate in terms of time (even to the minute level) and it also uses the local language for the hour and time.
Let me know what you think

```suggestion
    let lastUploadText = $derived(
    user.lastAssetUploadedAt
      ? (DateTime.fromISO(user.lastAssetUploadedAt).toRelative({ locale: $locale }) ?? $t('unknown_time'))
      : $t('never_uploaded'),
  );
```